tests/src/test/scala/cats/tests/EitherTests.scala

```diff
@@ -17,6 +18,11 @@ class EitherTests extends CatsSuite {
   val monad = implicitly[Monad[Either[Int, ?]]]
   val show = implicitly[Show[Either[Int, String]]]
 
+  val orderLaws = OrderLaws[Either[Int, String]]
+  checkAll("Either[Int, String]", orderLaws.eqv)
+  checkAll("Either[Int, String]", orderLaws.partialOrder(partialOrder))
+  checkAll("Either[Int, String]", orderLaws.order(order))
```

Comment:
Please see comment by one @adelbertc: https://github.com/non/cats/blob/master/tests/src/test/scala/cats/tests/ConstTests.scala#L27-L30

;)

This probably isn't quite as good as the `Const` approach of using types that _only_ have the instance for the laws being checked.